### PR TITLE
Use bloom filter to improve packet times

### PR DESCRIPTION
**Pull request by macpie**

### CODE REVIEW

**Comment by JayKickliter** [comment]
This PR is currently not cross-compilable. See [ch8782]

**Comment by JayKickliter** [comment]
https://github.com/Vonmo/erbloom/pull/10 laded, so this branch can now be cross-compiled to arm

File: src/state_channel/blockchain_state_channel_handler.erl [hunk 1]
```diff
@@ -65,10 +71,10 @@ send_offer(Pid, Offer) ->
     Pid ! {send_offer, Offer},
     ok.
 
--spec send_purchase(pid(), blockchain_state_channel_purchase_v1:purchase()) -> ok.
-send_purchase(Pid, Purchase) ->
+%-spec send_purchase(pid(), blockchain_state_channel_purchase_v1:purchase()) -> ok.
```
Comment (Vagabond) [inline src/state_channel/blockchain_state_channel_handler.erl:74]: Why commented out?

Comment (vihu) [reply src/state_channel/blockchain_state_channel_handler.erl]: Yeah, not sure what happened there, fixed

**Review by Vagabond — APPROVED** [review]

**Pull request merged**